pkg/cloud/cloud_test.go

```diff
@@ -84,6 +84,9 @@ func TestGetResources(t *testing.T) {
 	}, {
 		name:     "BareMetal resources are empty",
 		platform: configv1.BareMetalPlatformType,
+	}, {
+		name:     "AlibabaCloud resources are empty",
```

Comment:
This test should look more like the AWS or Azure tests above as we actually have resources to compare with